### DISCUSSION TITLE
GITHUB FOR NON PROFIT REQUEST

### DISCUSSION THREAD

**Discussion by iamlucas13** [community/community#110405] [Other Feature Feedback, Questions, & Ideas]
Good morning,

A few weeks ago I made a request to get GitHub for non profit. I have not heard back.

Does anyone have an idea of ​​the deadline?

Best.

Lucas

**Comment by ettaboyle** [comment]
Hi @LucasVOLET - I found [this information](https://docs.github.com/en/enterprise-cloud@latest/billing/managing-the-plan-for-your-github-account/discounted-plans-for-github-accounts#discounts-for-nonprofits-and-libraries) on GitHub Docs

> GitHub Enterprise Cloud provides free GitHub Team for organizations with unlimited private repositories, unlimited collaborators, and a full feature set to qualifying 501(c)3 (or equivalent) organizations and libraries. You can request a discount for your organization on [our nonprofit page](https://github.com/nonprofit).

Beyond that, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). We cannot respond to any account specific questions in our public community discussions.

**Reply by iamlucas13** [reply]
Hi,

Thanks you. Yes i followed this procedure. But i have any answer....

**Comment by anthonyroussel** [comment]
Hello.
Same for me unfortunately 😞
I submitted the non profit request form over a month ago now, and I still haven't heard back from GitHub.

**Reply by iamlucas13** [reply]
Hello, thanks for your answer

**Reply by it-wpc** [reply]
Hi all!

The same, submitted the nonprofit request over a month, not have any reply from GitHub ☹️

**Reply by anthonyroussel** [reply]
Hello! Have you received a response from Github since last month?
On my side, I still haven't had any messages from Github unfortunately ☹️

**Reply by it-wpc** [reply]
No... even asked them again about the status 😑
